python/hail/api2/table.py

```diff
@@ -766,7 +767,7 @@ def select(self, *exprs, **named_exprs):
         for k, e in named_exprs.items():
             all_exprs.append(e)
             analyze('Table.select', e, self._row_indices)
-            strs.append('`{}` = {}'.format(k, to_expr(e)._ast.to_hql()))
+            strs.append('{} = {}'.format(escape_id(k), to_expr(e)._ast.to_hql()))
```

Comment:
I'm not convinced the `drop` behavior is working. Can you add a test that drops the field `\`foo.a\``?

Reply:
added test for drop